src/main/java/org/fundacionjala/pivotal/core/api/APICommons.java

```diff
@@ -1,36 +1,46 @@
 package org.fundacionjala.pivotal.core.api;
 
 import com.jayway.restassured.response.Response;
-import java.util.HashMap;
-import java.util.Map;
+
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /**
  * APICommons.java
- * Class with utils method to execute an API request.
+ * Class with common method to execute an API request.
  */
 public final class APICommons {
-    private static Map<String, Response> requestResponse;
+    private static Map<String, Response> requestResponse = new HashMap<>();
     private static String endPoint;
+    private  APIResponseName date;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "date" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)